app/build.gradle

```diff
@@ -35,6 +35,7 @@ android {
     }
     lintOptions {
         disable 'InvalidPackage'
+        abortOnError false
```

Comment:
this can be removed. lint errors are fixed.

Reply:
Yeah i also thought the same. Had made the PR without this but there were still lint errors :confused: Let me try a re-travis build without this.

Reply:
I think it should work. Else go to `Analyze` -> `Inspect Code` and then manually lint fix errors if they have been introduced.

Reply:
@suresh-gandhi lint errors are still present : 
> Task :app:lint
Ran lint on variant release: 215 issues found
Ran lint on variant debug: 215 issues found

Reply:
let's see this build

Reply:
@jddeep no build errors on the development branch. Checked once again locally. Also, 313 works fine. 😕 I think they must have been mistakenly introduced in this branch or any other reason probably. Doesn't matter. Try fixing them and update the PR.

Reply:
Rebased. No lint errors in this branch too locally :confused: updated the PR but still it seems the lint errors remain here. What can be the problem now? :thinking:

Reply:
![Screenshot from 2019-06-20 20-33-02](https://user-images.githubusercontent.com/43133646/59859613-ab517600-939a-11e9-85f6-5241d05e0eb1.png)

Reply:
What to do now? :thinking: No errors. Should I use `abortonError false` for now. Maybe some problem with this branch over github :confused: ?

Reply:
@jddeep Yes, you can. I think, probably, it has something to do with the added dependency causing conflicts with the classes provided by Android. Would look into this today after I am finished up with the work in my hand.

Reply:
Yeah once this get's merged do have a check on your local machine too. The screenshot above is with the the dependency added only. Maybe some silly issue. Will update the PR to fix the build here for now. :+1:

Reply:
@suresh-gandhi Any thing more? We need to get this merged to progress.